src/commands/import.ts

```diff
@@ -1120,19 +1144,58 @@ command
 
       logger.info(`Created ${customFieldsToCreate.length} custom field(s)`);
 
-      logger.info('Checking if "Status" field is configured correctly...');
+      // At the time of writing this, the GraphQL mutation 'updateProjectV2Field' is only available on
+      // GitHub.com
+      const shouldConfigureStatusField = typeof gitHubEnterpriseServerVersion === 'undefined'
+      if (shouldConfigureStatusField) {
+        const sourceProjectStatusField = sourceProject.fields.nodes.find(field => field.name == "Status")!;
+        const sourceProjectStatusFieldOptions = sourceProjectStatusField.options!;
+
+        logger.info(`Configuring "Status" field: ${JSON.stringify(sourceProjectStatusFieldOptions)}`);
```

Comment:
I don't think we need to add so much verbose detail here - or if we want to we should only do it in verbose mode:

```suggestion
        logger.info('Configuring "Status" field options...');
```